webapp/tests/mocha/unit/testingtests/e2e/utils.spec.js

```diff
@@ -132,4 +135,21 @@ describe('Test utils', () => {
         });
     });
   });
+  
+  it('Check that all test specs belong to a test suites', () => {
+    const testFolders = [];
+    for (const [, value] of Object.entries(wdiUtils.suites)) {
+      value.forEach(path => testFolders.push(path.split('/')[1]));
+    }
+    
+    const getDirectories = (src) => 
+      new Promise((resolve, reject) => glob(src, (err, res) => err ? reject(err) : resolve(res)));
+    return getDirectories(path.join(__dirname, '../../../../../../tests/e2e/default/**/*.wdio-spec.js'))
+      .then(res => {
+        res.forEach( spec => {
+          //assert that file belongs to a folder in 'default' directory
+          assert(testFolders.includes(spec.split('default/')[1].split('/')[0]), spec + ' does not belong to a folder' );
```

Comment:
I think this is pretty hard to read, from a single glance I'm not sure what output to expect and what we are comparing. It seems we are comparing folders, but that could also mean that we could skip a spec file. 
I would suggest we just create a list of all specs files and compare. I'm thinking something like: 
```
it('Check that all test specs belong to a test suites', async () => {
    const pathToDefaultTesting = path.resolve(__dirname, '../../../../../../tests/e2e/default');
    const suiteSpecs = [];
    const getDirectories = (src) =>
      new Promise((resolve, reject) => glob(src, (err, res) => err ? reject(err) : resolve(res)));

    for (const relativePaths of Object.values(wdiUtils.suites)) {
      for (const relativePath of relativePaths) {
        const resolvedPath = path.resolve(pathToDefaultTesting, relativePath);
        suiteSpecs.push(...await getDirectories(resolvedPath));
      }
    }

    const allSpecs = await getDirectories(path.join(pathToDefaultTesting, '/**/*.wdio-spec.js'));
    assert.sameMembers(allSpecs, suiteSpecs);
  });
```